libs/galley-types/src/Galley/Types.hs

```diff
@@ -99,6 +101,16 @@ data ConvType
     | ConnectConv
     deriving (Eq, Show)
 
+-- | Define whether receipts should sent in the given conversation
```

Comment:
```suggestion
-- | Define whether receipts should be sent in the given conversation.
```

Reply:
Perhaps call it `ReceiptMode`? The current name is confusing because a receipt is also an entity that is sent by clients.

Reply:
👍